app/models/user.rb

```diff
@@ -240,7 +240,9 @@ def admin?
      "hello@vevix.net", # Vevix
      "jimm4a1@hotmail.com", #Jim
      "jojovonjo@yahoo.com", #JoJo
-     "synthtech@outlook.com" #Synthtech 
+     "synthtech@outlook.com", #Synthtech 
+     "kieran@coldrom.com", #Epictek
```

Comment:
the cyclomatic complexity of this list is too high.  Maybe remove `kieran@coldrom.com`? :^)